pyrinter/document.py

```diff
@@ -6,9 +6,11 @@
 from PIL import Image
 
 
-class Font(namedtuple("Font", ("font_name", "height"))):
+class Font(namedtuple("Font", ("font_name", "height", "weight"))):
     """
-    class to represent font properties (font name and height)
+    class to represent font properties (font name, height and weight)
+
+    weight can be either "normal" or "bold"
     """
```

Comment:
For backward compatibility, it would be beneficial to ensure that the `Font` class includes a default `weight` value. I suggest utilizing a `dataclass` for this purpose. Consider implementing it as follows:
```python
@dataclass
class Font:
    """
    class to represent font properties (font name, height and weight)
    """

    font_name: str
    height: int
    weight: Literal["normal", "bold"] = "normal"
```

Reply:
Yes, much better idea 👍